.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
```

Comment:
I don't actually have much of a preference one way or another for [this rule](https://eslint.org/docs/rules/arrow-parens). I only bring it up because React uses a lot of event handlers like
```jsx
<div onClick={event => doSomething(event.target)}` />
```
and you may find that cleaner than
```jsx
<div onClick={(event) => doSomething(event.target)}` />
```
in which case turning it off would be a good idea. Otherwise I'll delete this line.